### ISSUE TITLE
[BUG] Speed Selector unable to change between Knots and Mach

### ISSUE THREAD

**Issue by Odinsonhagen** [flybywiresim/a32nx#2605]
**v0.5.0**

**Describe the bug**

**To Reproduce**

1. Plane departed with a provided MCDU setup
2. During the climbing, Auto pilot changes Knots speed parameter into Mach speed.
3.It doesn't effect anything about flying but MACH speed parameter cannot be switched into KNOTS.

**Expected behavior**

Knots speed parameter should not be changed by itself or it must be switched if it is needed. In this case, the airplane changes Knots speed parameter into MACH speed and it cannot be fixed. 

Was this working before/when did the issue start occurring?

It doesnt happen all the time. 4 out of 10 flight would  a precise ratio I can give.

Is this a problem in the vanilla unmodded game?

not sure but I have never seen something like this before.

**Comment by marcsoler** [comment]
Not sure if this helps. But today I was able to reproduce this bug by switching from AP1 to AP2. I then had the speed indicator suddenly switched to mach and wasn't able to switch back to knots.

**Comment by Watsi01** [comment]
This is due to the latest Asobo update. It only switches automatically - in CLB when reaching mach .78, in DESC when passing FL260. We didn't find a way to make it work manually too so far.

**Comment by bisinchi** [comment]
I think a side effect of that bug is what I experienced with yesterday's downloaded dev mod. I managed to change to speed by selecting a lower mach than the cruze mach. But somehow, the mach was still being used inside the AP as when the selected airspeed was reached, the associated mach became again the selected value. As the plane went down, the AP obviously tried to maintain a higher an higher speed. Had to reselect the speed every once in a while until transition altitude, then no problem after.
